<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubtractionConfigFrame</class>
 <widget class="QFrame" name="SubtractionConfigFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="firstNumberLabel">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;First Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="minNumberLabel">
       <property name="text">
        <string>Min Number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="minNumberLineEdit">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The smallest allowed value for the first number in the subtraction. It must be smaller than (or equal to) the maximum number provided.

This can be negative, and include any number of decimal places. The generated number will have as many digits after the decimal point as the most supplied between the minimum and maximum values.

The answer will contain as many digits after the decimal place as the most used by the first and second numbers.</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="maxNumberLabel">
       <property name="text">
        <string>Max Number</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="maxNumberLineEdit">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The largest allowed value for the first number in the subtraction. It must be larger than (or equal to) the minimum number provided.

This can be negative, and include any number of decimal places. The generated number will have as many digits after the decimal point as the most supplied between the minimum and maximum values.

The answer will contain as many digits after the decimal place as the most used by the first and second numbers.</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lastNumberLabel">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Second Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="secondMinLabel">
       <property name="text">
        <string>Min Number</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="secondMinLineEdit">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The smallest allowed value for the second number in the subtraction. It must be smaller than (or equal to) the maximum number provided.

This can be negative, and include any number of decimal places. The generated number will have as many digits after the decimal point as the most supplied between the minimum and maximum values.

The answer will contain as many digits after the decimal place as the most used by the first and second numbers.</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="secondMaxLabel">
       <property name="text">
        <string>Max Number</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="secondMaxLineEdit">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The largest allowed value for the second number in the subtraction. It must be larger than (or equal to) the minimum number provided.

This can be negative, and include any number of decimal places. The generated number will have as many digits after the decimal point as the most supplied between the minimum and maximum values.

The answer will contain as many digits after the decimal place as the most used by the first and second numbers.</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="largestNumberFirstLabel">
       <property name="text">
        <string>Largest Number First</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="largestNumberFirstCheckBox">
       <property name="toolTip">
        <string>If checked, whatever numbers are generated will be presented with the largest number as the first number, and the smallest one as the second number. Otherwise, what ever is generated will be used without modification.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
